spec_analysis/jw8_redshift_fitting_solutions.ipynb

```diff
@@ -25,7 +25,9 @@
     "    - Get the library of spectra\n",
```

Comment:
<p><code>min_redshift</code>, <code>max_redshift</code>, <code>delta_redshift</code> and <code>z_values</code> are never actually used... it might be easier to follow by commenting out the first block, placing it after defining the fixed redshift, and naming <code>z_values</code> and <code>zz</code> the same (so that just uncommenting would result in the other behavior).&nbsp; Or just clarify the comment at the end to be more explicit (something like "change zz to z_values to use a list of redshifts").</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
Done